### PR TITLE
Added demo for lookup component

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-demo](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-demo)

File: src/app/docs/lookup/lookup-docs.component.html [hunk 1]
```diff
@@ -8,4 +8,44 @@
...
+    class="lookup-demo-form"
+    novalidate
+    style="width: 50%;"
+    [formGroup]="myForm"
+  >
+    <div
+      class="sky-form-group"
+    >
+      <label
+        id="my-friends-label"
```
Comment (Blackbaud-SteveBrush) [inline src/app/docs/lookup/lookup-docs.component.html:22]: `skyId`?

File: src/app/public/plugin-resources/code-examples/lookup/basic/lookup-demo.component.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  class="lookup-demo-form"
+  novalidate
+  [formGroup]="myForm"
+  (ngSubmit)="onSubmit()"
+>
+  <div
+    class="sky-form-group"
+  >
+    <label
+      id="my-friends-label"
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/plugin-resources/code-examples/lookup/basic/lookup-demo.component.html:11]: `skyId`?

File: src/app/docs/lookup/lookup-docs.component.html (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/docs/lookup/lookup-docs.component.html]: Good catch. Fixed.

File: src/app/public/plugin-resources/code-examples/lookup/basic/lookup-demo.component.html (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/plugin-resources/code-examples/lookup/basic/lookup-demo.component.html]: Good catch. Fixed.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-demo](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-demo)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Review by blackbaud-johnly — APPROVED** [review]
The only thing I saw is that the current field label goes against the guidance at https://developer.blackbaud.com/skyux/design/guidelines/form-design#field-usability not to use questions as field labels. This was already in place, so I think we can move forward with it as is and ask @Blackbaud-ToddRoberts if we should revisit the label later.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-demo](https://host.nxt.blackbaud.com/skyux-lookup?_sp.spa=pr-lookup-demo)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**